### ISSUE TITLE
Add pyDataverse to README.md

### ISSUE THREAD

**Issue by skasberger** [IQSS/dataverse-client-python#53]
Just a question: would it make sense, to mention pyDataverse somewhere in the README?

**Comment by pdurbin** [comment]
@skasberger yes! I had the same thought. Please go ahead and make a pull request if you are so inclined!
